staging/proj.finassistant/manifest.tt

```diff
@@ -0,0 +1,139 @@
+// Copyright 2016-2020 The Board of Trustees of the Leland Stanford Junior University
+//
+// Redistribution and use in source and binary forms, with or
+// without modification, are permitted provided that the following
+// conditions are met:
+//
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above
+//    copyright notice, this list of conditions and the following
+//    disclaimer in the documentation and/or other materials
+//    provided with the distribution.
+// 3. Neither the name of the copyright holder nor the names of its
+//    contributors may be used to endorse or promote products derived
+//    from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+// FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+// COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+// INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+// (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+// HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+// OF THE POSSIBILITY OF SUCH DAMAGE.
+
+class @proj.finassistant
+#_[thingpedia_name="Finance Assistant"]
+#_[thingpedia_description="Research assistant leveraging Alpha Vantage APIs to help users quickly find relevant financial data to support their investing decisions"]
+#[subcategory="media"]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[website="https://www.alphavantage.co"]
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none(api_key="VPJXG30AV7OQAM22");
```

Comment:
Don't put the API key publicly like this! Make this `api_key=$?` and create a secrets.json file containing the real API key (`{"api_key": "..."}`). Do not commit that file. Send me the key privately and then I will commit an encrypted copy of that file.

(Amend this commit and force-push to remove the key from git history)